ccloud/docs/replicator-to-cloud-configuration-types.rst

```diff
@@ -0,0 +1,211 @@
+.. _replicator-to-cloud-configurations:
+
+|crep-full| to |ccloud| Configurations
+======================================
+
+Whether you are migrating from on-prem to cloud or have a persistent "bridge to cloud" strategy, you can use |crep-full| to copy Kafka data to |ccloud|.
+Learn the different ways to configure |crep| and |kconnect-long|.
+
+.. figure:: images/replicator-to-ccloud.png
+
+===============
+Concepts Review
+===============
+
+Before diving into the different ways to configure |crep|, let's first reprise some basic concepts regarding |crep| and |kconnect-long|.
+This will help you understand the logic for configuring |crep| because how the |kconnect-long| cluster is configured will affect how |crep| should be configured.
+
+|crep| is a Kafka connector and runs on Connect workers.
+Even the :ref:`Replicator executable <replicator_executable>` has a bundled Connect worker inside.
+
+|crep| has an embedded consumer that reads data from the origin cluster, and the Connect worker has an embedded producer that copies that data to the destination cluster, which in this case is |ccloud|.
+To configure the proper connection information for |crep| to interact with the origin cluster, use the prefix ``src.``.
```

Comment:
These rules apply when running in distributed mode, could be helpful to specify the differences in configuration if the user is deploying in executable mode.

Reply:
@skaundinya15 this is certainly open to discussion but just wanted to share the logic behind not showing Replicator executable and showing just distributed.

1. The examples are Docker-based and thus use the Replicator Docker image, so that is the distributed mode.
2. Showing both distributed mode version and executable version would pollute the docs page, which is already pretty long with these 4 config types

Perhaps in parallel, what if there were a separate docs page that guided users how to take a distributed config and convert it to executable?

Reply:
That's a good point, a docs page on how to take a distributed config and converting it to an executable config is probably the best move here.